plugins/chat/spec/system/mention_warnings_spec.rb

```diff
@@ -0,0 +1,61 @@
+# frozen_string_literal: true
+
+describe "when a mention won't create a notification", type: :system, js: true do
```

Comment:
So in my current branch I created a `JIT mentions` spec, which could be used for this. For now I suggest we just go with: `Rspec.describe "Mentions warnings"`